test/functional/deprecated_rpc.py

```diff
@@ -19,5 +19,11 @@ def run_test(self):
         self.log.info("Using -deprecatedrpc=estimatefee bypasses the error")
         self.nodes[1].estimatefee(1)
 
+        # test validateaddress deprecation
+        dep_validate_address = self.nodes[0].validateaddress("mnvGjUy3NMj67yJ6gkK5o9e5RS33Z2Vqcu")
+        assert("ismine" not in dep_validate_address)
```

Comment:
Is it enough to check `ismine` key? Maybe merge `nodes[0].validateaddress` with `nodes[0].getaddressinfo` and that should contain `nodes[1].validateaddress`?

Reply:
It's enough to determine that they are returning different things as one should have `ismine` and the other not.